src/utils/axiosInterceptors.ts

```diff
@@ -6,9 +6,14 @@ const EXPIRY_HEADER_KEY = 'Cache_Expires';
 const EXPIRES_IN_SECONDS = 60 * 30;
 const DELAY = 3000;
 const RETRIES = 5;
-export interface RequestConfig extends AxiosRequestConfig {
-  useCache?: boolean;
-  retries?: number;
+
+// we are extending axios' AxiosRequestConfig:
+// https://stackoverflow.com/questions/58777924/axios-typescript-customize-axiosrequestconfig
+declare module 'axios' {
+  export interface AxiosRequestConfig {
```

Comment:
VS Code says the `AxiosRequestConfig` in `import axios, { AxiosRequestConfig } from 'axios'; is not used.
Lint doesn't say anything. I tried without importing `AxiosRequestConfig` it also works. 

Don't know what to do here. Should it be removed from the import?

Reply:
Nice catch! Yes, it should be removed, because we are not really using it - interesting that linter didn't complain. Thanks, fixed.

Reply:
Nice. Resolving.